src/index.js

```diff
@@ -6,11 +6,12 @@ var React = require('react');
 var omit = require('lodash.omit');
 var mediaQuery = require('./mediaQuery');
 var toQuery = require('./toQuery');
-var matchMedia = typeof window !== 'undefined' ? window.matchMedia : null;
+var matchMedia = require('matchmedia');
 
 var defaultTypes = {
   component: React.PropTypes.func,
-  query: React.PropTypes.string
+  query: React.PropTypes.string,
+  values: React.PropTypes.object
```

Comment:
can you define the shape of this object with React.PropTypes.shape? also need docs on this new prop